pkg/strategy/fixedmaker/inventory_skew.go

```diff
@@ -0,0 +1,70 @@
+package fixedmaker
+
+import (
+	"context"
+
+	"github.com/c9s/bbgo/pkg/bbgo"
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+var (
+	zero = fixedpoint.Zero
+	two  = fixedpoint.NewFromFloat(2.0)
+)
+
+type InventorySkewBidAskRatios struct {
+	bidRatio fixedpoint.Value
+	askRatio fixedpoint.Value
+}
+
+// https://hummingbot.org/strategy-configs/inventory-skew/
+// https://github.com/hummingbot/hummingbot/blob/31fc61d5e71b2c15732142d30983f3ea2be4d466/hummingbot/strategy/pure_market_making/inventory_skew_calculator.pyx
+type InventorySkew struct {
+	Session *bbgo.ExchangeSession
+	Market  types.Market
+
+	InventoryRangeMultiplier fixedpoint.Value `json:"inventoryRangeMultiplier"`
+	TargetBaseRatio          fixedpoint.Value `json:"targetBaseRatio"`
+}
+
+func (s *InventorySkew) Initialize(session *bbgo.ExchangeSession, market types.Market) error {
+	s.Market = market
+	s.Session = session
+	return nil
+}
+
+func (s *InventorySkew) CalculateBidAskRatios(ctx context.Context, quantity fixedpoint.Value) (*InventorySkewBidAskRatios, error) {
```

Comment:
maybe we can make it not to depend on session? because it could make the test harder to write.

how about just pass base balance, quote balance and the current price of base?

Reply:
or maybe you can keep this InventorySkew, and pull out the formula to a function?

Reply:
I would like to make it not to depend on session.